ci/scripts/multi-arch-docker.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+date="$(date +%Y%m%d)"
+ghcraddr="ghcr.io/singularity-data"
+
+components=(
+  "risingwave"
+  "compute-node"
+  "meta-node"
+  "frontend-node"
+  "compactor-node"
+)
+
+echo "--- ghcr login"
+echo "$GHCR_TOKEN" | docker login ghcr.io -u "$GHCR_USERNAME" --password-stdin
+
+for component in "${components[@]}"
```

Comment:
Would be better if we can merge the two scripts into one. e.g.

```
docker.sh --merge
docker.sh --build
```

... if you are a bash master 🤣